include/Game/Events/EReset.h

```diff
@@ -0,0 +1,16 @@
+#ifndef Reset_h__
+#define Reset_h__
+
```

Comment:
I believe we usually do ``Namespace_FileName_h__``. This would be ``Events_Reset_h__``.